### ISSUE TITLE
Scroll freeze/lag gmail app (AMP-email)

### ISSUE THREAD

**Issue by claesbengtsson** [ampproject/amphtml#34701]
### Description

Hi, this is my first Bug report so bear with me. We currently have an issue when sending an email to the Gmail APP for iOS. We have created an AMP mail with an interactive menu, where content gets showed depending on what menu-item you click. Unfortunately when interacting with one of the menu-items the email scroll tends to get stuck. The weirdest part about this issue is that it seemingly happens randomly. Sometimes the email can be scrolled fine, otherwise not regardless if we have made any changes to the code or not. When it gets stuck on my phone, it also gets stuck on my colleagues phone, and when it works for me it also has worked for him. (See video and you'll see what I mean). Really frustrating to reproduce the error! 😣

_(The design of the email is not finished, I'm fully aware the items etc aren't aligned)_
https://user-images.githubusercontent.com/82397663/120776951-0dbec500-c525-11eb-810a-f6b7202fbc1d.mp4

I’ve tried searching Stack Overflow/reddit & google to see if someone has bumped into this issue before, but can’t seem to find anything. I’ve also tried removing parts of the code step by step to try finding the bug/cause. However the “lag/freeze” still seem to happen randomly.

Has anyone experienced this issue before? Or something that reminds of it? Basically I’m struggling to recreate the issue and to identify the cause of it.

Best regards
Claes

### Reproduction Steps

1. Paste the AMP code into your esp
2. Send the email to your gmail, and open the gmail in the gmail app, (iPhone)
3. The scroll freeze should occur when selecting an item from the menu and then trying to vertically scroll. It happens randomly, sometimes it works fine just to malfunction an hour later.

If you are willing to look into this I'll send over the code!

### OS(s) Affected

iOS

### Device(s) Affected

iPhone Xr, iPhone X, iPhone 11 (+More device-models probably).

**Comment by claesbengtsson** [comment]
1. This only occurs on the gmail app on phone iOS. Just tried https://mail.google.com on my phone, and it doesn't recognize AMP so it shows HTML version. Not sure if this is what you meant.

2. Sure, I've attached it.
[test-menus-bsft-4.html.zip](https://github.com/ampproject/amphtml/files/6617385/test-menus-bsft-4.html.zip)

**Comment by alanorozco** [comment]
> Just tried https://mail.google.com on my phone, and it doesn't recognize AMP so it shows HTML version.

Try it on Desktop.

**Comment by claesbengtsson** [comment]
Yes this I have tried before, on desktop it works without any issues. I've only run into this issue on gmail app iOS mobile

**Comment by claesbengtsson** [comment]
@alanorozco Any update on this? :)

**Comment by UldisPF** [comment]
Have noticed the same buggy behaviour. Tested on various iOS devices - around 12 of them. Iphones, iPads both show the same behaviour (that scrolling sometimes gets stuck) when AMP email is viewed on gmail app. On android and desktop clients that support AMP this behaviour isnt seen. 

Code example where issue is noticed: 
[testAMP.html.zip](https://github.com/ampproject/amphtml/files/9652545/testAMP.html.zip)

**Comment by bdjang** [comment]
is there any update on this issue? we have an AMP email currently sending and many have encountered this same scrolling issue on the Gmail app.

here are my device and app specs:
iPhone XR, iOS 15.7
Gmail App Version 6.0.221016.1574978

fwiw, it seems that changing the iPhone orientation to landscape and back to portrait while the AMP email is opened “unfreezes” and allows you to scroll the AMP email without issue. see screen recording below: first few seconds the scroll is stuck, after switching to landscape/portrait, scroll becomes unstuck

![wapo-amp-email-scroll-issue](https://user-images.githubusercontent.com/6575035/200881677-49b940bf-2a7e-4e32-b17c-8ae8a5e47e53.gif)

**Comment by dimified** [comment]
Has anyone fixed this issue yet? Seems like it is not resolved yet.

**Comment by UldisPF** [comment]
A bit unsure if google cares enough for AMPhtml to be adopted. This bug
exists for years.
